### PR TITLE
Change Plan decommissioning 

### PR DESCRIPTION
**Pull request by clairecfu**
image
- [x] also fixed bug where power ports and network connections dont get copied over to AssetCP on decommission. We had solved this for asset modify, so used same logic as that to fix that bug on decommission

### CODE REVIEW

File: rackcity/views/change_plan_views.py [hunk 1]
```diff
@@ -138,7 +138,25 @@ def change_plan_remove_asset(request, id):
...
+                asset_cp_object.delete()
+            else:
+                for field in asset_cp.related_asset._meta.fields:
+                    if (
+                            field.name != "id"
+                            and field.name != "assetid_ptr"
+                            and field.name != "chassis"
+                    ):
+                        setattr(asset_cp, field.name, getattr(asset_cp.related_asset, field.name))
+
```
Comment (clairecfu) [inline rackcity/views/change_plan_views.py:157]: if a chassis has blades with changes on the change plan -> dont actually remove it from AssetCP, just make it the same as live .

File: rackcity/views/change_plan_views.py [hunk 2]
```diff
@@ -423,7 +441,7 @@ def change_plan_execute(request, id):
         updated_asset_mappings[asset_cp] = updated_asset
         differs_from_live = True
 
-        if created:
+        if created and not asset_cp.is_decommissioned:
```
Comment (clairecfu) [inline rackcity/views/change_plan_views.py:444]: counting bug

File: rackcity/views/change_plan_views.py (hunk 1 above)
Comment (juliasaveliff) [reply rackcity/views/change_plan_views.py:157]: this is super fun

File: rackcity/views/change_plan_views.py (hunk 2 above)
Comment (juliasaveliff) [reply rackcity/views/change_plan_views.py:444]: it be like. that

**Pull request merged**